src/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.label.searchbar {
+  width: 100%;
+  height: 70px;
+}
+
+.searchbar--big {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;

```

Comment:
Move `border-radius`, `border` and `box-shadow` props to the `input` selector. 
Your form will have no size or any visual styles. It should be just a pure wrapper.